### ISSUE TITLE
This repo is too large to be easily cloned.  Are there any other download hosts or torrents?

### ISSUE THREAD

**Issue by Defunctionalize** [Unity-Technologies/UniversalRenderingExamples#8]
Im having considerable trouble getting ahold of the assets I need to complete [this URP course](https://learn.unity.com/tutorial/custom-render-passes-with-urp) from my current location.  You cant download only part of a git repo, so I'm stuck trying and failing to grab the whole thing.  This repo should be broken up or download mirrors or torrents should be provided

**Comment by wheee** [comment]
I agree that the different examples shoudl be separted to their own repos.  There's too much bloat in these examples.

**Comment by wilde-eep** [comment]
Why is this example project 1Gb is size? You are making it difficult to access for many people who would like to see the contents. Please think about the size of your assets when making example projects in future. :-/

**Comment by thygrrr** [comment]
To be honest, 1GB exceeds the recommended repo size for a git repo and will strain what github tolerates. (Patchy LFS support no withstanding)
